File: perl/fizzbuzz.pl
use strict;
use warnings;

use constant FIZZ => 'Fizz';
use constant BUZZ => 'Buzz';
use constant STOP => 100;
use constant EOL => "\n";

for (my $i = 1; $i <= STOP; $i++) {
  if ($i % 15 == 0) {
    print FIZZ . BUZZ . EOL;
  } elsif ($i % 3 == 0) {
    print FIZZ . EOL;
  } elsif ($i % 5 == 0) {
    print BUZZ . EOL;
  } else {
    print $i . EOL;
  }
}

File: ruby/fizzbuzz.rb
FIZZ = 'Fizz'
BUZZ = 'Buzz'
STOP = 100

for i in 1..STOP do
	if i % 15 == 0 then
		puts FIZZ + BUZZ
	elsif i % 3 == 0 then
		puts FIZZ
	elsif i % 5 == 0 then
		puts BUZZ
	else
		puts i
	end
end
